src/components/_plans/Calendar/components/CalendarHeader/components/CurrentView/CurrentView.tsx

```diff
@@ -19,13 +19,19 @@ const CurrentView: FC<Props> = ({ isMonthsView, isYearsView, month, year }) => {
   const renderViewContent = () => {
     if (!isMonthsView && !isYearsView) {
       return (
-        <>
-          {new Intl.DateTimeFormat('en-US', { month: 'long' }).format(new Date(0, month))}
-          <span className={classes.calendarViewYear}>{year}</span>
-        </>
+        <button className={classes.calendarView}>
+          <span className={classes.month}>
+            {new Intl.DateTimeFormat('en-US', { month: 'long' }).format(new Date(0, month))}
```

Comment:
Maybe think about moving it to some custom format function so it will be easier later to add multi languagte support